presto-main/src/main/java/io/prestosql/server/testing/TestingPrestoServer.java

```diff
@@ -229,6 +233,8 @@ public TestingPrestoServer(
                     binder.bind(TestingEventListenerManager.class).in(Scopes.SINGLETON);
                     binder.bind(AccessControlManager.class).to(TestingAccessControlManager.class).in(Scopes.SINGLETON);
                     binder.bind(EventListenerManager.class).to(TestingEventListenerManager.class).in(Scopes.SINGLETON);
+                    binder.bind(PrestoGroupProviderManager.class).in(Scopes.SINGLETON);
+                    newOptionalBinder(binder, GroupProviderManager.class).setBinding().to(TestingGroupProviderManager.class);
```

Comment:
Since we're getting rid of the `GroupProviderManager` interface and the optional binder, this can be
```java
binder.bind(GroupProvider.class).toInstance(user -> ImmutableSet.of());